### PR TITLE
Add Crab Smart Chain

### PR DESCRIPTION
**Pull request by hujw77**

### CODE REVIEW

File: src/util/index.ts [hunk 1]
```diff
@@ -74,6 +74,7 @@ export async function lookupBlock(
     }
     let high = lastBlock.number;
     let low = extraParams?.chain === "terra" ? 4724001 : 0;
+    low = extraParams?.chain === "crab" ? 4969901 : 0;
```
Comment (0xngmi) [inline src/util/index.ts:77]: lowest block on crab chain is 4969901?

Comment (0xngmi) [reply src/util/index.ts:77]: also please rewrite this with an object

Comment (0xngmi) [reply src/util/index.ts:77]: well i can do that myself, nvm

Comment (hujw77) [reply src/util/index.ts:77]: > lowest block on crab chain is 4969901?

Yes, The genesis block of crab smart chain is 4969901.

Comment (hujw77) [reply src/util/index.ts:77]: > well i can do that myself, nvm

Yes, I think you do that better.

**Comment by 0xngmi** [comment]
added support in 55e5c39

**Pull request closed without merging**